modules/primer-buttons/stories.js

```diff
@@ -2,6 +2,9 @@ import React from 'react'
 import { storiesOf } from '@storybook/react'
 
 storiesOf('Button', module)
+  .add('button', () => (
+    <button>Button</button>
```

Comment:
What's the purpose of this here? Wouldn't it be better if we put stories in `primer-base`?